package/MDAnalysis/core/groups.py

```diff
@@ -317,9 +317,94 @@ class _ImmutableBase(object):
     __new__ = object.__new__
 
 
+
+def _only_same_level(function):
+    @functools.wraps(function)
+    def wrapped(self, other):
+        if not isinstance(other, (ComponentBase, GroupBase)):  # sanity check
+            raise TypeError("Can't perform '{}' between objects:"
+                            " '{}' and '{}'".format(
+                                function.__name__,
+                                type(self).__name__,
+                                type(other).__name__))
+        if self.level != other.level:
+            raise TypeError("Can't perform '{}' on different level objects"
+                            "".format(function.__name__))
+        if self.universe is not other.universe:
+            raise ValueError(
+                "Can't operate on objects from different Universes")
+        return function(self, other)
+    return wrapped
+
+
 class GroupBase(_MutableBase):
     """Base class from which a Universe's Group class is built.
 
+	Instances of :class:`GroupBase` provide the following operations that
```

Comment:
I was definitely going to include these operations in #1190 which is the tutorial/introduction docs.  But yeah, adding them to GroupBase is effectively hiding them from users